spring-cloud-dataflow-rest-client/src/main/java/org/springframework/cloud/dataflow/rest/client/config/DataFlowClientProperties.java

```diff
@@ -86,6 +88,26 @@ public void setAuthentication(Authentication authentication) {
 		 */
 		private String accessToken;
 
+		/**
+		 * OAuth2 Client Id.
+		 */
+		private String clientId;
+
+		/**
+		 * OAuth2 Client Secret.
+		 */
+		private String clientSecret;
+
+		/**
+		 * OAuth2 Token Uri.
+		 */
+		private String tokenUri;
+
+		/**
+		 * OAuth2 Scopes.
+		 */
+		private Set<String> scope;
```

Comment:
nitpicky should this be scopes?

Reply:
Annoyingly it's like this in spring security settings as well. Took a same key.